etabotsite/etabotsite/settings.py

```diff
@@ -51,7 +51,7 @@
     logging.warning('cannot load custom_settings.json due to "{}"'.format(
         e))
 CUSTOM_SETTINGS = custom_settings
-
+PROD_HOST_URL = prod_host_url
```

Comment:
why creating an extra variable here instead of directly using `prod_host_url`

Reply:
capitalized letters are needed for Django framework to parse settings